llvm/include/llvm/Analysis/LoopAccessAnalysis.h

```diff
@@ -365,17 +365,25 @@ class MemoryDepChecker {
   void mergeInStatus(VectorizationSafetyStatus S);
 
   struct DepDistanceStrideAndSizeInfo {
+    // Strides could either be scaled (in bytes, taking the size of the
+    // underlying type into account), or unscaled (in indexing units; unscaled
+    // stride = scaled stride / size of underlying type). Here, strides are
+    // unscaled.
     const SCEV *Dist;
-    uint64_t StrideA;
-    uint64_t StrideB;
+    uint64_t MaxStride;
```

Comment:
nit: move to `MaxStride`?

```suggestion
    /// Strides could either be scaled (in bytes, taking the size of the
     // underlying type into account), or unscaled (in indexing units; unscaled
     /// stride = scaled stride / size of underlying type). Here, strides are
     /// unscaled.
    uint64_t MaxStride;
```

Reply:
Doesn't the comment also apply to CommonStride? That's the reason I put it on top.

Reply:
It applies to all strides, but not `Dist`, maybe separate the comment from Dist with a newline (and a newline after the strides)?